frontend/app/onboard.tsx

```diff
@@ -0,0 +1,245 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Make sure to call the POST /users route at the end to add the user to our database.